pkg/platform/model/checkpoints.go

```diff
@@ -27,8 +27,9 @@ type Checkpoint []*mono_models.Checkpoint
 
 // Language represents a language requirement
 type Language struct {
-	Name    string `json:"name"`
-	Version string `json:"version"`
+	Name            string `json:"name"`
+	Version         string `json:"version"`
+	ResolvedVersion string `json:"resolved_version,omitempty"`
```

Comment:
This doesn't belong here imo. ResolvedVersion is a concern only of the language runner as written, so this type should exist only there. You could inherit this struct though.

Reply:
Okay, I've removed it.